packages/react/npm/jsx-runtime.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+if (process.env.NODE_ENV === 'production') {
+  module.exports = require('./cjs/react/jsx-runtime.production.min.js');
```

Comment:
This needs to say `react-jsx-runtime`, not `react/jsx-runtime`. Same below.

Reply:
Oops, I missed this one. Good call! :D